coding_setup.md

```diff
@@ -42,16 +42,17 @@ Squadbox uses Django with a MySQL backend (you can replace with any other backen
 * configure your relay_server (postfix or something else) in config/settings.py
 * use port other than 25 (default is currently set at 587)
 
-#### setup the database 
+#### setup the database
 * (optional: only during new database setup) change root password by: `set PASSWORD = PASSWORD('newPassword');`
 * `mysql -u root -p`
 * `create database squadbox;`
-* Give privileges to the user that will access the database from django: `grant all privileges ON squadbox.* TO admin@localhost;`
+* Give privileges to the user that will access the database from django: `create user 'admin'@'localhost' identified by 'admin';` `grant all privileges ON squadbox.* TO 'admin'@'localhost';`
 
 #### install schema and create superuser
 * `python manage.py syncdb`and create superuser
 * Convert schema app to be managed by South: `python manage.py schemamigration schema --initial`
 * Then do fake migration:  `python manage.py migrate schema 0001 --fake`
+* If error involving caching_sha2_password is encountered, use sqlite3 instead
```

Comment:
I would change to "If you encounter errors getting django to work with mysql, you can use alternative databases such as sqlite3."